scripts/bin/infra

```diff
@@ -1,4 +1,10 @@
 #!/bin/bash
+
+# Check for the flag SLANG_INFRA_RELEASE before calling `set -u`
```

Comment:
With using default substitution, to replace a non-existing variable with an empty string (like what you are doing below), not sure why would it fail?
Moving that statement after `set -u` would work correctly.